core/tests/puppeteer-acceptance-tests/utilities/common/puppeteer-utils.ts

```diff
@@ -82,7 +86,7 @@ export class BaseUser {
 
     const headless = process.env.HEADLESS === 'true';
     const mobile = process.env.MOBILE === 'true';
-    const specName = process.env.SPEC_NAME;
+    this.specName = process.env.SPEC_NAME ?? '';
```

Comment:
Why is this change needed?

Reply:
Because we are using `specName` in new `setupVideoRecording` method.

Reply:
I declared `specName` attribute as string. So, we are required to have `?? ''`, as `process.env.SPEC_NAME` can be `null` (different type than string).

Reply:
OK, maybe worth explaining in the code then.

Though perhaps better to fail noisily if it's null?

Reply:
It reverted back to original while moving implementation of screen recording from its own method to openBrowser method.